catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -130,7 +130,7 @@ const WorkDetails: FunctionComponent<Props> = ({ work }: Props) => {
 
   const authService =
     (video && getMediaClickthroughService(video)) ||
-    (audio && getMediaClickthroughService(audio));
+    (audio?.[0] && getMediaClickthroughService(audio[0]));
```

Comment:
Do we know that if there are multiple pieces of audio, they'll always have the same auth service?

e.g. we wouldn't have three bits of audio, but you need to click through to listen to the second?

Reply:
That's a good point, and I meant to ask @aray-wellcome about this. I would imagine so.

Reply:
If you're asking if we might have different access statuses on bits of audio, the answer is yes. We might have 1 open audio track but two restricted for example, because of what they contain specifically.